plugins/MultiSites/vue/src/AllWebsitesDashboard/SitesTableSite.vue

```diff
@@ -33,15 +33,20 @@
       </template>
     </td>
 
-    <td><span class="value">{{ site.nb_visits }}</span></td>
-    <td><span class="value">{{ site.nb_pageviews }}</span></td>
-    <td><span class="value">{{ site.hits }}</span></td>
-    <td v-if="displayRevenue"><span class="value">{{ site.revenue }}</span></td>
+    <td><span class="value">{{ formatNumber(site.nb_visits) }}</span></td>
+    <td><span class="value">{{ formatNumber(site.nb_pageviews) }}</span></td>
+    <td><span class="value">{{ formatNumber(site.hits) }}</span></td>
+    <td v-if="displayRevenue">
+      <span class="value">{{ formatCurrency(site.revenue, site.currencySymbol) }}</span>
```

Comment:
This needs an additional check for `site.isGroup` now, or we fall back to the symbol as an empty string.

If that hidden special feature is used there is no `currencySymbol` available, displaying as `undefined123345`.